### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by BenTheElder**
I fixed https://github.com/kind-ci/containerd-nightlies to publish 1.6.x properly and the branch name https://github.com/kind-ci/containerd-nightlies/commits/main, but builds in the base image need fixing now, will finish later:

> #34 0.678 sha256sum: /tmp/containerd.sha256: no properly formatted SHA256 checksum lines found

### CODE REVIEW

File: images/base/Dockerfile [hunk 1]
```diff
@@ -27,19 +27,19 @@ ARG TARGETARCH
...
 ARG CONTAINERD_BASE_URL="https://github.com/kind-ci/containerd-nightlies/releases/download/containerd-${CONTAINERD_VERSION}"
 ARG CONTAINERD_URL="${CONTAINERD_BASE_URL}/containerd-${CONTAINERD_VERSION}.linux-${TARGETARCH}.tar.gz"
-ARG CONTAINERD_AMD64_SHA256SUM="84c7ec4beec0e83aa02a9e1891acd6a8404675207f1e00163227fb2c82ed1e1c"
-ARG CONTAINERD_ARM64_SHA256SUM="e70d705b6063fd7facebbf6b106b4a078787da23ba6a2248b9047a5f1d0bc6d7"
-ARG CONTAINERD_PPC64LE_SHA256SUM="970fd22e9cff65a1c2ae02db1655388df12dcdebc421967d3cfd1a8932bdfdc7"
-ARG CONTAINERD_S390X_SHA256SUM="94005c30c29f24b13a977dd38283b814cc596e13dd6dde4dc289dd5c0654800e"
+ARG CONTAINERD_AMD64_SHA256SUM="Not"
+ARG CONTAINERD_ARM64_SHA256SUM="Not"
+ARG CONTAINERD_PPC64LE_SHA256SUM="Not"
+ARG CONTAINERD_S390X_SHA256SUM="Not"
```
Comment (BenTheElder) [inline images/base/Dockerfile:36]: Well that would be why 🙄

File: images/base/Dockerfile [hunk 2]
```diff
@@ -27,19 +27,19 @@ ARG TARGETARCH
...
 # The repository contains latest stable releases and nightlies built for multiple architectures
-ARG CONTAINERD_VERSION="1.5.10"
+ARG CONTAINERD_VERSION="1.6.1"
 ARG CONTAINERD_BASE_URL="https://github.com/kind-ci/containerd-nightlies/releases/download/containerd-${CONTAINERD_VERSION}"
-ARG CONTAINERD_URL="${CONTAINERD_BASE_URL}/containerd-${CONTAINERD_VERSION}.linux-${TARGETARCH}.tar.gz"
-ARG CONTAINERD_AMD64_SHA256SUM="84c7ec4beec0e83aa02a9e1891acd6a8404675207f1e00163227fb2c82ed1e1c"
-ARG CONTAINERD_ARM64_SHA256SUM="e70d705b6063fd7facebbf6b106b4a078787da23ba6a2248b9047a5f1d0bc6d7"
-ARG CONTAINERD_PPC64LE_SHA256SUM="970fd22e9cff65a1c2ae02db1655388df12dcdebc421967d3cfd1a8932bdfdc7"
-ARG CONTAINERD_S390X_SHA256SUM="94005c30c29f24b13a977dd38283b814cc596e13dd6dde4dc289dd5c0654800e"
+ARG CONTAINERD_URL="${CONTAINERD_BASE_URL}/containerd-${CONTAINERD_VERSION}-linux-${TARGETARCH}.tar.gz"
```
Comment (BenTheElder) [inline images/base/Dockerfile:32]: 1.6.0+ names `-` instead of `.` here

**Comment by aojea** [comment]
do we want to do this before code freeze? 
containerd bump scares me :grimacing: 
Should we wait until k/k master gets more stable to not introduce many changes at same time, the go1.18 thing still needs to soak and settle

**Comment by BenTheElder** [comment]
containerd has a bad CVE so we need to bump at least to latest 5.X

code freeze != test freeze, and we run all the matching CI here for a reason.

In fact we've had no flakes on this PR where we usually have flakes these days.

most of these changes like the go version and toml and tooling bumps only affect our own development not Kubernetes

**Comment by aojea** [comment]
> code freeze != test freeze, and we run all the matching CI here for a reason.

give it at least one or two days more so the CI can digest go1.18 , so we can observe patterns, there are a lot of new flakes , the problem are the flakes with low rate, those are not likely going to be caught here

**Comment by BenTheElder** [comment]
> give it at least one or two days more so the CI can digest go1.18 , so we can observe patterns, there are a lot of new flakes , the problem are the flakes with low rate, those are not likely going to be caught here

on the contrary, we should ship sooner, code freeze is the dedicated period in which we fix observed issues, we should introduce any potential issues _before_ code freeze

**Comment by BenTheElder** [comment]
And again, kind, kops, cluster-api, etc are not bound by code freeze.

**Comment by BenTheElder** [comment]
building 1.6.2 with manual nightly trigger https://github.com/kind-ci/containerd-nightlies/actions/workflows/main.yml

**Comment by BenTheElder** [comment]
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kind/2691/pull-kind-conformance-parallel-ga-only/1507119208016646144
/retest

**Comment by BenTheElder** [comment]
Should be ready.

**Comment by aojea** [comment]
/lgtm
:crossed_fingers:

**Pull request merged**